docs/digger/admin/managing-digger-logs-artifacts.asciidoc

```diff
@@ -0,0 +1,31 @@
+== Managing Digger Build Logs and Artifacts
+App builds started by Digger produce build logs and build artifacts.
+Over time, these will build up. There are a number of options to manage the buildup. If you are using Digger with RHMAP, these options can be specified
+by setting environment variables in the `millicore` deployment(???) configuration in
```

Comment:
@pmacko1 for p0: yes. It could be applied to the upstream project. for q1, let's use environment variables.